docker/bookkeeper/entrypoint.sh

```diff
@@ -74,6 +75,27 @@ create_bookie_dirs() {
   done
 }
 
+# Create a Bookie ID if this is a newly added bookkeeper pod
+# or read the Bookie ID if it already exists
+set_bookieid() {
+  IFS=',' read -ra directories <<< $1
+  BK_ID_DIR="${directories[0]}"
+  BK_ID_FILE="${BK_ID_DIR}/id"
+  if [ `find ${BK_ID_FILE} | wc -l` -gt 0 ]; then
+    echo "Found BookieID"
+    BK_bookieId=`cat ${BK_ID_FILE}`
+  else
+    echo "BookieID not found"
+    if [[ $BK_setNewBookieID == "true" ]];then
+      BK_bookieId="`hostname -s`-`date "+%Y%m%d-%H%M%S"`"
```

Comment:
I remember that it was suggested to use a UUID instead of timestamp here, right? It should not change much the purpose of the change, though.

Reply:
Have incorporated this change